shared/constants/hardware-wallets.js

```diff
@@ -47,4 +47,4 @@ export const TRANSPORT_STATES = {
   VERIFIED: 'VERIFIED',
   DEVICE_OPEN_FAILURE: 'DEVICE_OPEN_FAILURE',
   UNKNOWN_FAILURE: 'UNKNOWN_FAILURE',
-};
+}
```

Comment:
The changes in this file are no longer needed, can be rectified by adding the `;` and new line back to the file.